cmd/cache/main.go

```diff
@@ -41,6 +42,7 @@ func main() {
 	flag.UintVar(&expiryDelay, "expiry-delay", 30, "The delay in days before deleting an unused CachedImage.")
 	flag.IntVar(&proxyPort, "proxy-port", 8082, "The port where the proxy is listening on this machine.")
 	flag.StringVar(&ignoreNamespace, "ignore-namespace", "kuik-system", "The address the probe endpoint binds to.")
+	flag.StringVar(&registry.Endpoint, "registry-endpoint", "kuik-registry:5000", "The address of the registry where cached images are stored.")
```

Comment:
Should be named it 'registry-service' ? I thought we target a Service and it should be explained?

Reply:
I think it's better to keep it named `registry-endpoint` since it sets the variable `registry.Endpoint`. And actually it is not a service name but a real endpoint since it includes a port.

Additionally, in the case we would some day allow to run the registry outside of the cluster like [asked on twitter](https://twitter.com/Qasim_Sarfraz/status/1620396306584993796) by Qasim Sarfaz, we really want to speak about an endpoint rather than a service. To do so, we actually only need to update the chart and allow to set this flag with a value (and eventually add authentication and a flag to set `registry.Protocol` as well).